tensorboard/webapp/metrics/views/card_renderer/scalar_card_component.ts

```diff
@@ -244,4 +251,17 @@ export class ScalarCardComponent<Downloader> {
         this.isProspectiveFobFeatureEnabled)
     );
   }
+
+  toggleTableExpanded() {
+    // Manually resizing an element sets a style value on the element which takes
+    // precedence over any classes the element may have. This value must be removed
+    // for the table to expand or collapse correctly.
+    if (this.dataTableContainer) {
+      this.dataTableContainer.nativeElement.style.height = '';
```

Comment:
So you cannot add a new class with height: unset;?

Reply:
Unfortunately not, inline styles will always override those provided by a class.